pkg/sql/schema_changer_test.go

```diff
@@ -1245,6 +1245,49 @@ func dropIndexSchemaChange(
 	}
 }
 
+// TestDropColumn tests that dropped columns properly drop their Table's CHECK constraints
+func TestDropColumn(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	params, _ := tests.CreateTestServerParams()
+
+	s, sqlDB, kvDB := serverutils.StartServer(t, params)
+	defer s.Stopper().Stop(context.TODO())
+
+	if _, err := sqlDB.Exec(`
+CREATE DATABASE t;
+CREATE TABLE t.test (k INT PRIMARY KEY, v INT CHECK (v >= 0), a INT DEFAULT 0 CHECK (a <= v), b INT default 100 CHECK (b > a) );
```

Comment:
Can you wrap to multiple lines?

```
CREATE DATABASE t;
CREATE TABLE t.test (
  k INT PRIMARY KEY,
  v INT CONSTRAINT check_v CHECK (v >= 0),
  a INT DEFAULT 0 CONSTRAINT check_av CHECK (a <= v),
  b INT DEFAULT 100 CONSTRAINT check_ab CHECK (b > a)
);
```

I've also named the constraints; see below for details.